src/routes/synapse/room_events.rs

```diff
@@ -123,11 +124,14 @@ pub async fn room_event_handler(
         (logged_in_user, token)
     };
 
+    let body_message = body.body.as_deref();
```

Comment:
Wondering if we should change this to this version and work with an `Option<String>` instead. What do you think @jmoguilevsky @lauti7 @agusaldasoro ?
```suggestion
    let body_message = body.body.clone();
```

Reply:
I'm not sure I understood, it's already an Option<String>

Reply:
`Option<String>`  vs `Option<&str>`

Reply:
- [String vs &str; in Rust functions](https://web.archive.org/web/20201112023149/https://hermanradtke.com/2015/05/03/string-vs-str-in-rust-functions.html)
- [Creating a Rust function that accepts String or &str;](https://web.archive.org/web/20201112021433/https://hermanradtke.com/2015/05/06/creating-a-rust-function-that-accepts-string-or-str.html)